### PR TITLE
1254460: Added authors.txt and displayed it in About dialog

### PR DESCRIPTION
**Pull request by vritant**

### CODE REVIEW

**Comment by Ceiu** [comment]
Do we want to standardize how nicknames/aliases are specified in there? We have at least three different formats: First "alias" last, first last (alias) and first (alias) last.

**Comment by barnabycourt** [comment]
Standardizing would be good.

**Comment by vritant** [comment]
@Ceiu I figured we should stick to authors' chosen format ( I copied the names from git commits ). going to standardize now.

**Comment by vritant** [comment]
@Ceiu lol :) fixed, thanks :)

**Comment by vritant** [comment]
retest this please

**Comment by alikins** [comment]
If folks have strong opinions they can override me, but I'd rather not have to keep the info updated.

(The dialog already has long standing outdated info in it like the copyright dates and the 'Website', I'd
rather not add more. If that could be updated as part of building releases, that would be better)

File: src/subscription_manager/gui/about.py [hunk 1]
```diff
@@ -34,6 +34,7 @@
             "Red Hat trademarks are not licensed under GPLv2. No permission is "
             "granted to use or replicate Red Hat trademarks that are incorporated "
             "in this software or its documentation.\n")
+AUTHORS = [line.strip() for line in open("%s/docs/authors.txt" % os.getcwd(), 'r')]
```
Comment (alikins) [inline src/subscription_manager/gui/about.py]: This only works if you are running subscription-manager-gui from the source checkout.

```
[subscription-manager (vritant/1254460-credits u=)]$ cd
[~]$ sudo subscription-manager-gui 
Traceback (most recent call last):
  File "/sbin/subscription-manager-gui", line 132, in <module>
    from subscription_manager.gui import managergui
  File "/usr/share/rhsm/subscription_manager/gui/managergui.py", line 50, in <module>
    from subscription_manager.gui.about import AboutDialog
  File "/usr/share/rhsm/subscription_manager/gui/about.py", line 37, in <module>
    AUTHORS = [line.strip() for line in open("%s/docs/authors.txt" % os.getcwd(), 'r')]
IOError: [Errno 2] No such file or directory: '/home/adrian/docs/authors.txt'
```

Comment (alikins) [inline src/subscription_manager/gui/about.py]: AUTHORS isn't a constant here so ALLCAPS is a little bit misleading.
(Though we aren't particular consistent about that, namely the odd CFG in a few modules)

Comment (alikins) [inline src/subscription_manager/gui/about.py]: relpaths can also be kind of weird, especially when running as root.

```
[~]$ cd
[~]$ mkdir docs
[~]$ ln -s /etc/shadow- docs/authors.txt
[~]$ sudo subscription-manager-gui 
```

sub-man-abouts

**Comment by alikins** [comment]
NACK

**Comment by vritant** [comment]
@Ceiu @alikins  I'm caught between standardizing names and working on something that automatically updates authors ( or at least suggest to add author name from a pre-push hook if the author does not exist )
to implement both these requirements can make things complicated, something has to give.

open to suggestions!

**Comment by vritant** [comment]
Giving up on this, untill we can figure out a better way to do it.

**Comment by barnabycourt** [comment]
Hi, A manual authors.txt is a great starting point. Automation can be added later if required.

**Pull request closed without merging**